custom/echis_reports/ucr/reports/hew_performance_report_focal_person.json

```diff
@@ -0,0 +1,221 @@
+{
+  "domains": [
+    "l10k2020ethiopia",
+    "fmoh-echis-staging"
+  ],
+  "server_environment": [
+    "production",
+    "echis"
+  ],
+  "report_id": "static-hew_performance_focal_person",
+  "data_source_table": "static-hew_performance_indicators",
+  "config": {
+    "description": "HEW performance report for Focal Person",
+    "title": "HEW performance report (Focal Person)",
+    "aggregation_columns": [
+      "health_center",
+      "health_post",
+      "received_on",
+      "hew_name"
+    ],
+    "filters": [
+      {
+        "datatype": "string",
+        "choice_provider": {
+          "include_descendants": true,
+          "type": "location",
+          "order_by_hierarchy": true,
+          "show_full_path": false
```

Comment:
@snopoke I'm hoping to merge the properties `order_by_hierarchy` and `show_full_path`.  That is, in my estimation, locations should always be ordered alphabetically when the path isn't displayed, and they should always be arranged in tree order when the full path _is_ displayed.

The echis reports only use `show_full_path = false`, but half of them have `order_by_hierarchy = true`.  Do you know if this was a deliberate decision, and if so, what the rationale for that is?

Reply:
@karmuno , @emord mentioned that you may know more about this. Happy to offline if that's easier.

Reply:
I don't know anything about those. @karmuno might be a better person to answer your question.